Source/WebKit/UIProcess/API/glib/WebKitWebView.cpp

```diff
@@ -4757,9 +4757,9 @@ struct ViewSaveAsyncData {
     RefPtr<API::Data> webData;
     GRefPtr<GFile> file;
 };
+#if ENABLE(MHTML)
```

Comment:
I do think `ViewSaveAsyncData` belongs behind this also.

I see `webkit_web_view_save_finish()` uses this type not behind a guard, but that doesn't make sense since `webkit_web_view_save()` is guarded, so it will never work.

Reply:
I agree with your comments, and have updated the code so that when `MHTML` is not enabled it does `g_task_return_boolean(task, FALSE);`. 

I was tempted to return an error, but seeing that errors need registering, I am not sure it's worth as `MHTML` will remain enabled outside experiments. Do you think that's acceptable?